crates/bevy_tasks/src/task_pool.rs

```diff
@@ -233,7 +238,7 @@ impl TaskPool {
     ///     });
     /// }
     ///
-    pub fn scope<'env, F, T>(&self, f: F) -> Vec<T>
+    pub fn scope<'env, F, T>(&self, thread_executor: Option<Arc<ThreadExecutor>>, f: F) -> Vec<T>
```

Comment:
This seems like a general usability regression. Is there a way to separate this out?